embeddings/utils/loggers.py

```diff
@@ -17,3 +20,45 @@ def get_logger(name: str, log_level: Union[str, int] = DEFAULT_LOG_LEVEL) -> log
     logger.propagate = False
 
     return logger
+
+
+@dataclass
+class LightningLoggingConfig:
```

Comment:
Could we make this class more general? Not just for config purposes? And add `log_artifacts` / `log` functionality if needed? Would it be time consuming? Because in current approach is pretty hard to add another logger, as it is stricly hardcoded to use csv / wandb / tensorboard and it's just config.

Reply:
I have added `WandbWrapper` as a separated class. Actually, adding another logger is quite hard and should be carefully tested in the pytorch lightning framework, so it should be strict.

Reply:
I understand it, but I would like it to be more generic as Lighting logging is more broader than only Mlflow or Tensorboard https://pytorch-lightning.readthedocs.io/en/stable/common/loggers.html

Reply:
Lets focus on finishing this version and we can think about second version of these loggers.